project/src/components/app/app.tsx

```diff
@@ -1,5 +1,13 @@
-function App(): JSX.Element {
-  return <p>Hello, world!</p>;
+import Main from '../../pages/main';
+
+type AppScreenProps = {
+  offers: number;
+}
+
+function App({offers}: AppScreenProps ): JSX.Element {
+  return (
+    <Main offersCount = {offers}/>
```

Comment:
```suggestion
    <Main offersCount={offers} />
```

название offers соответствует названию коллекции. Как-то не сходится, число и коллекция

Reply:
++